src/pcapi/flask_app.py

```diff
@@ -80,7 +80,7 @@ def log_request_details(response: flask.wrappers.Response) -> flask.wrappers.Res
         "method": request.method,
         "route": str(request.url_rule),  # e.g "/offers/<offer_id>"
         "path": request.path,
-        "queryParams": request.query_string.decode("UTF-8"),
+        "queryParams": request.query_string.decode(request.url_charset, errors="replace"),
```

Comment:
Pourrait-on plutôt utiliser
```suggestion
        "queryParams": request.query_string.decode(request.url_charset, errors="backslashreplace"),
```
Pour avoir une idée de la séquence originale remplacée ?